_dashboards/get-started/quickstart-dashboards.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Quickstart for OpenSearch Dashboards
+nav_order: 20
+has_children: false
+---
+
+# Quickstart for OpenSearch Dashboards
+
+This quickstart covers the core concepts for getting started with OpenSearch Dashboards. You'll learn how to:
+
+- Add sample data
```

Comment:
Do we also need to include a section on guiding users to add their own data? so not just sample data

Reply:
Based on our chat, we'll  leave the quickstart with sample data. A separate "Adding data" section about adding your own data will be created.